### DISCUSSION TITLE
Hotkeys and Improved Accessibility for Visually Impaired Users

### DISCUSSION THREAD

**Discussion by DevSmoger** [stoatchat/discussions#771] [Feature Suggestions]
Dear Revolt.chat Development Team,

I hope this message finds you well. I am writing to request the addition of hotkeys functionality similar to what Discord offers, as well as improvements to button labeling for better accessibility.

A friend of mine, who is visually impaired, relies on screen readers such as NVDA and JAWS to navigate and use applications. While we've experimented with Revolt.chat, we've encountered difficulties due to the lack of keyboard shortcuts and insufficient labeling on buttons. Currently, when using Tab to navigate, the screen reader only announces "button" without any further context, which makes it challenging to use the app effectively.

By adding hotkeys and assigning meaningful labels to the buttons, Revolt.chat could become significantly more accessible to users who rely on screen readers and keyboard navigation. These changes would make the application far more user-friendly for those with visual impairments and would help expand its accessibility to a wider audience.

We believe that with these enhancements, Revolt.chat could provide a much better experience for visually impaired users. Please let us know if this request is something you could consider for future updates.

Thank you for your time and consideration.

**Comment by 1enify** [comment]
As far as I understand Revolt team is already working on injecting keybinds into the app, if this is what you assume. But since new web app development has been put off, it may take some time to release the new features to the public.

**Reply by DevSmoger** [reply]
That's great news, and we’ll definitely be looking forward to it! I couldn't find any information on this previously, and since we are new to the Revolt.chat app, I wanted to highlight these issues. I understand how extensive and time-consuming development can be. We’ll be excited to try out the upcoming updates and provide feedback once they're available.
Best regards.
